File: exception_handling.dart
import 'dart:io';

void main() {
  try {
    stdout.write("Enter a number: ");
    int num = int.parse(stdin.readLineSync()!);

    if (num == 0) {
      throw Exception("Number should not be zero");
    }

    print("Valid Number: $num");
  } catch (e) {
    print("Error: $e");
  } finally {
    print("Execution completed");
  }
}

File: php/Exception Handling.php
<?php
try {
    $num = 10;
    if ($num < 20) {
        throw new Exception("Number is less than 20");
    }
    echo "Number is OK";
}
catch (Exception $e) {
    echo "Error: " . $e->getMessage();
}
finally {
    echo "<br>Execution completed";
}
?>